apps/condo/domains/division/utils/helpers.ts

```diff
@@ -0,0 +1,30 @@
+// import { DivisionWhereInput } from "@condo/schema"
+import { get } from 'lodash'
+import { DivisionWhereInput } from '../../../schema'
+import { IFilters } from '../../contact/utils/helpers'
+
+export const searchToQuery = (search?: string): DivisionWhereInput[] => {
+    if (!search) {
+        return
+    }
+
+    return [
+        { properties_some: { address_contains_i:  search } },
+    ]
+}
+export const filtersToQuery = (filters: IFilters): DivisionWhereInput => {
```

Comment:
Please refer to this manual (https://doma.atlassian.net/wiki/spaces/SBER/pages/787808257/1) after this (https://github.com/open-condo-software/condo/pull/498) PR will come to master...
I guess you will need to create a custom filter for properties _some, but for address `getStringContainsFilter('address')` will be enough